index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Team 8</title>
+</head>
+<body>
+  
```

Comment:
Hey Cinthia, thanks for doing this but this was actually my task (as assigned in our project kanban board, and issues. This was discussed during today's call... We really want you to work with us, but you have to communicate with the team.

Reply:
No problem, Alissa! I didn't see the index file, so I created it only to test if I could do a PR without a mess.